src/qnetvo/ansatz_library.py

```diff
@@ -140,3 +144,106 @@ def pure_phase_damping(noise_params, wires):
 
     ry_setting = 2 * math.asin(math.sqrt(noise_params[0]))
     qml.ctrl(qml.RY, control=wires[0])(ry_setting, wires=wires[1])
+
+
+class two_qubit_depolarizing(Channel):
+    """Applies a two-qubit depolarizing channel using Kraus operators.
+
+    The channel is called as a quantum function ``two_qubit_depolarizing(gamma, wires)``
+
+    :param gamma: The amount of colored noise in the channel.
+    :type gamma: Float
+
+    :param wires: Two wires on which to apply the colored noise.
+    :type wires: qml.Wires
+
+    For a noise parameter :math:`\\gamma`, the two-qubit depolarizing
+    noise model is represented by the following function:
+
+    .. math::
+
+        \\mathcal{N}(\\rho) = (1-\\frac{16}{15}\\gamma)\\rho + (\\frac{16}{15})(\\frac{\\gamma}{4})) \\mathbb{I}
+
+    :raises ValueError: if ``0 <= gamma <= 1`` is not satisfied.
+    """
+
+    num_params = 1
+    num_wires = 2
+    grad_method = "F"
+
+    @classmethod
+    def _kraus_matrices(cls, *params):
```

Comment:
In the latest versions of PL we have changed to `compute_kraus_matrices()`, so this may come up as an issue when updating the PL version, but I think it's fine for now.

Reply:
It will, but we'll be in dependency Hell and forced to update everything to maintain proper working if `compute_kraus_matrices` is used. Would rather just add it to the tab of 0.21/0.22 updates